docs/instrumentation-list.yaml

```diff
@@ -2082,19 +2082,52 @@ libraries:
           type: LONG
   couchbase:
   - name: couchbase-2.0
+    description: |
+      This instrumentation enables database CLIENT spans and metrics for Couchbase 2.0 operations. It automatically traces key-value operations (get, upsert, replace, remove), view queries, N1QL queries, and cluster management operations.
     source_path: instrumentation/couchbase/couchbase-2.0
     scope:
       name: io.opentelemetry.couchbase-2.0
     target_versions:
       javaagent:
       - com.couchbase.client:java-client:[2,3)
-    configurations:
-    - name: otel.instrumentation.couchbase.experimental-span-attributes
```

Comment:
this removal was intentional. while the 2.0 instrumenter has the attributeextractor, it only works within the 2.6 instrumentation, so it made sense to me to remove it from being documented here